webapp/apps/taxbrain/models.py

```diff
@@ -768,25 +769,26 @@ class TaxSaveInputs(models.Model):
     # Creation DateTime
     creation_date = models.DateTimeField(default=datetime.datetime(2015, 1, 1))
 
-
-    @property
-    def tax_result(self):
+    def get_tax_result(self):
         """
         If taxcalc version is greater than or equal to 0.13.0, return table
         If taxcalc version is less than 0.13.0, then rename keys to new names
         and then return table
         """
         outputurl = OutputUrl.objects.get(unique_inputs__pk=self.pk)
         taxcalc_vers = outputurl.taxcalc_vers
-        taxcalc_vers = tuple(map(int, taxcalc_vers.split('.')))
-        if taxcalc_vers >= (0, 13, 0):
-            return self._tax_result
+        # only the older (pre 0.13.0) taxcalc versions are null
+        if taxcalc_vers:
+            taxcalc_vers = LooseVersion(taxcalc_vers)
         else:
-            return rename_keys(self._tax_result, PRE_TC_0130_RES_MAP)
+            return rename_keys(self.tax_result, PRE_TC_0130_RES_MAP)
 
-    @tax_result.setter
-    def tax_result(self, result):
-        self._tax_result = result
+        # older PB versions stored commit reference too
+        # e.g. taxcalc_vers = "0.9.0.d79abf"
+        if taxcalc_vers >= LooseVersion("0.13.0"):
```

Comment:
See comment above... Might also be a good idea to make a `utils.py` file for this commit anyway?

Reply:
Ok, I moved this method to the bottom of `taxbrain/helpers.py`. The utility and helper functions need to be refactored and re-organzied. See issue #816